src/cubism-common/MotionManager.ts

```diff
@@ -230,20 +388,55 @@ export abstract class MotionManager<Motion = any, MotionSpec = any> extends util
         }
 
         let audio: HTMLAudioElement | undefined;
+        let analyzer: AnalyserNode | undefined;
+        let context: AudioContext | undefined;
 
         if (config.sound) {
+            const isBase64Content = sound && sound.startsWith('data:audio/wav;base64');
```

Comment:
```suggestion
            const isBase64Content = sound && sound.startsWith('data:');
```

Testing the `data:` prefix would be enough.

Reply:
Wait what?! I probably did that. Sorry, missed that one maybe.😥

Reply:
Wait what?! I probably did that. Sorry, missed that one maybe.😥